# Aims

## Determine the Availability of Environmental DNA in a Controlled Environment

- Design primers for amplifying a fat-tailed dunnart marker region.
    - Find short ~30 bp mitochondrial marker that distinguishes the fat-tailed dunnart from common contaminants (human, chicken, pig) and available marsupial reference genomes, and is not present in its nuclear genome (not a NUMT).
    - Test primers in silico.
    - Run a PCR with the primers on DNA extracted from tissue, confirm amplification of the segment, and send for Sanger sequencing to confirm fat-tailed dunnart origin.
- Determine if tile swaps, bottom substrate, or scat can all provide fat-tailed dunnart DNA.
    - Collect samples from a fat-tailed dunnart cage and apply [DNA Shield](https://zymoresearch.eu/collections/dna-rna-shield?srsltid=AfmBOoq3Yeb8XjGjqSAzWHl-lxgnBJTvxZbW7I6KqkBD6GVdFpXV_Um9) for preservation. Store at room temperature.
    - Extract DNA using soil or fecal specific extraction kits.
    - Run PCR reaction, confirm amplification of marker region, and send for Sanger sequencing. Include positive and negative controls (DNA from tissue sample and solution with water instead of DNA).
- Test for how long fat-tailed dunnart DNA can be detected in a sample left under a tile in the System Garden.
    - Swap the tile, scat, or substrate - depending on the results of the previous experiment - and collect, preserve, PCR amplify and sequence a sample every week until fat-tailed dunnart DNA is no longer detected.
    - Do a control experiment on a roof tile that has not been in contact with a fat-tailed dunnart.

When sampling, record the ambient temperature and surface temperature of the roof tile and bottom substrate or soil.

Consider designing primers for a loci with a different lenght in dunnart than other species (a repeat region), so succesful DNA amplification can be tested with a gel electrophoresis rather than sequencing.

## Isolate Environmental DNA from Wild Fat-Tailed Dunnarts

- Isolate fat-tailed dunnart DNA from underneath roof tiles in the field.
    - From underneath roof tiles with evidence of dunnart occurance, collect swap sample, soil, and scat and preserve it with DNA Shield.
    - Collect field blanks from underneath roof tiles with no evidence of dunnart occurance.
    - Extract DNA and confirm or deny dunnart pressence by PCR amplification and sequencing of the marker region.
    - Compare the successrate of the method in grassland and dessert.

## Compare the Diet of Fat-Tailed Dunnarts in Different Environments.

- Build reference genome database of fat-tailed dunnart prey.
    - Evaluate the geographic range overlap with Victoria and taxonomic coverage of publically available insect genomes on Genbank.
    - Extent the database by sequecing insects caught in pitfall traps at our field sites.
- Design primers for metabarcoding assay of scat.
    - Design primers for metabarcoding assay of scat that can delineate insects by family.
    - Test the primers on a mixture of equimolar DNA extracted directly from insects.
- Describe the diet of fat-tailed dunnarts in grassland and dessert.
    - Characterize and compare the diet of fat-tailed dunnarts in grassland and dessert using the metabarcoding assay.
    - Complement the assay with information about the difference in insect species pressence between the two habitats.
