# Non-Invasive Monitoring of the Fat-Tailed Dunnart
This repository holds the [TIGRR Lab](https://tigrrlab.science.unimelb.edu.au/) project expanding the fat-tailed dunnart (*Sminthopsis crassicaudata*) model system to include non-invasive methods of describing its distribution and ecological interactions in the wild.

## Table of Contents
1. [Motivation](#motivation)
2. [Background](#background)
3. [Aims](#aims)
    -  [Non-Invasive Detection of Fat-tailed Dunnart](#non-invasive-detection-of-fat-tailed-dunnart).
    - [Minimal-Intervention Description of Fat-Tailed Dunnart Diet](#minimal-intervention-description-of-diet)

## Motivation

Australia's rich and unique biodiversity.
- [Woinarski et al. (2015)](https://doi.org/10.1073/pnas.1417301112)

Biodiversity loss.
- [Woinarski et al. (2019)](https://doi.org/10.1016/j.biocon.2019.108261)

Conservation efforts.

- [Kearney et al. (2018)](https://doi.org/10.1017/S0030605317001739)

Fauna surveys.

Environmental DNA for identifying rare or elusive species.

- [Franklin et al. (2019)](https://doi.org/10.1016/j.biocon.2018.11.006)

The fat-tailed dunnart as a model organism.


The phylogeny of the fat-tailed dunnart.


The ecology of the fat-tailed dunnart.

- [Scicluna et al. (2021)](https://doi.org/10.1071/ZO21014)

The conservation status of the fat-tailed

Habitat North and south of the Murray River.

- [Fenner et al. (1934)](https://doi.org/10.2307/209495)

Difference in diet between habitats.

Flooding and subsequent decline of the population south of the river.

Current techniques for identifying the fat-tailed dunnart and determining its diet.

Environmental DNA can provide a simpler and faster method of identification.

Can we tell how many dunnarts visited a site? [^1].

- [de Flamingh et al. (2023)](https://doi.org/10.3389/fgene.2022.1021004)

Can we tell if the dunnarts in the north eat more hard insects than soft insects compared to the south?

We can describe when then animal was present in an area through time by sampling sediment deposits at different depths.

- [Barnosky et al. (2017)](https://doi.org/10.1126/science.aah4787)

Isolating dunnart DNA from the environment increases the range within which we can assess the genetic variation of populations to areas where we cannot capture the animals themselves.

- [Pedersen et al. (2021)](https://doi.org/10.1016/j.cub.2021.04.027)

[^1]: How about pooling the scat from a tile grid?

## Background
About eDNA [barcodes, qPCR, metabarcoding, metagenomics (shot-gun sequencing)]


[the invention of large public databases, nanopore sequencing,  airDNA sampling]

Back in time - Ancient DNA

The dunnarts ecology and phylogeny (relatedness and rarity and family)
[]

The dunnart in the TIGRR Lab
[]
Millie Thesis

Immunity and Olfaction

Alignment with TIGRR Lab Goals
[]
## Aims

- Non-invasive detection of fat-tailed dunnart in different environments
- Minimal-intervention description of what a dunnart has eaten

## Non-Invasive Detection of Fat-Tailed Dunnart
### Background

Saumya Wanniarachchi - forest fire effect on diet of small mammals

### Approach

[From feces to coprolites]

[Mouth swaps]

## Minimal-Intervention Description of Fat-Tailed Dunnart Diet
### Background

Capturing dunnarts. Identifying scat.

Thermal Cameras. When there is no evidence. 

### Approach

Swapping tiles.

Burrows. eDNA from the air.

## Notes
Parts of this README was created with the use of ChatGPT 4.0.

## References
